### PR TITLE
Eric vook Albuquerque

### PR DESCRIPTION
**Pull request by EricVook**
Please review my proposed hometown recommendations.

@mburriss @warrenburnside @kristinawalker

### CODE REVIEW

**Comment by buchhei** [comment]
@EricVook Looks like you made a mistake in line 3

File: EricVook-Albuquerque.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Albuquerque, NM
+
+### Great places to eat
+
+
+### Fun things to do
+
+Visit Sandia Peak
```
Comment (buchhei) [inline EricVook-Albuquerque.md]: Looks like you made a mistake in line 3.

Comment (mburriss) [inline EricVook-Albuquerque.md]: One of these days I'll make it west.  :)

**Comment by EricVook** [comment]
Resolves #70

File: EricVook-Albuquerque.md (hunk 1 above)
Comment (EricVook) [inline EricVook-Albuquerque.md]: > Looks like you made a mistake in line 3.

I think it ok, but I will make it a sub-heading which will probably look better.  Good comment, @buchhei .

**Comment by EricVook** [comment]
I just changed a file, partly to fix the comment by @buchhei .

**Comment by EricVook** [comment]
Merging at this time so I can make web changes.

If anyone is not happy, I will entertain further comments. :smile:

**Pull request merged**